### ISSUE TITLE
not working in touch 2.3.*

### ISSUE THREAD

**Issue by lokibyte** [elmasse/Ext.ux.Cover#5]
Hi ,

i'm getting "[DEPRECATE][Ext.ux.Cover#constructor] setOrientation() is deprecated" when i used in sencha touch 2.3.0 can help me to do solve this issue please..

**Comment by lokibyte** [comment]
Hi Maximiliano Fierro

Please do asap , thanks for your needfull reply

Regards
Loki

On Mon, Mar 17, 2014 at 7:02 PM, Maximiliano Fierro <
notifications@github.com> wrote:

> Hi @loki4u https://github.com/loki4u, sorry for the delayed response,
> somehow I did not received this issue notification. I'll take a look and
> let you know.
> 
> ## 
> 
> Reply to this email directly or view it on GitHubhttps://github.com/elmasse/Ext.ux.Cover/issues/5#issuecomment-37814960
> .

## 

"Small Goal is CRIME"

**Comment by elmasse** [comment]
I might need more info on this. I have tested it using sencha CDN version 2.3.0 and 2.3.1 (see link below) and I'm not getting any message in the console.

http://extjs.cachefly.net/touch/sencha-touch-2.3.1/sencha-touch-all.js

Apart from that, the orientation property is a private one I have added to Cover so I can  apply a custom css on orientation changes.

**Comment by elmasse** [comment]
Sorry I didn't mean to close the issue. Reopening

**Comment by lokibyte** [comment]
carousel is not showing properly in sencha touch 2.3.1 when i click on any image it shown in weired manner..

**Comment by elmasse** [comment]
@loki4u I have been testing latest version on iPad simulator. Unfortunately I cannot reproduce the issue you mentioned. When taping on any image I got the following in the console:

```
[Log] itemtap [ (demo.js, line 46) Object, 2,Object,Object,Object,Object, Object]
```

Please if you can create a small test zip and send it over I will look at it. Or post more information on how to reproduce it, maybe modifying the demo in the repo.

Here are the screenshots:

![screen shot 2014-03-18 at 8 51 04 pm](https://f.cloud.github.com/assets/223235/2455155/d1d54ce8-aef8-11e3-9d77-83f52cb1d33d.png)

![screen shot 2014-03-18 at 8 51 47 pm](https://f.cloud.github.com/assets/223235/2455156/d65f40e8-aef8-11e3-86fd-e822a75214d3.png)

**Comment by elmasse** [comment]
Fixed in version v0.2.5

**Comment by lokibyte** [comment]
Hi Maximiliano Fierro

Thanks for you patience and thankfull feedback! :-)

Regards
Loki

On Thu, Mar 20, 2014 at 4:46 AM, Maximiliano Fierro <
notifications@github.com> wrote:

> Closed #5 https://github.com/elmasse/Ext.ux.Cover/issues/5.
> 
> ## 
> 
> Reply to this email directly or view it on GitHubhttps://github.com/elmasse/Ext.ux.Cover/issues/5
> .

## 

"Small Goal is CRIME"
